CHANGES.md

```diff
@@ -0,0 +1,10 @@
+1.1.0 / ??-??-2017
+==================
+
+* Add PDF output support
+* Add support to customize the output format with a `format` request parameter
+
+1.0.0 / ??-??-2017
```

Comment:
Maybe add accurate date here, it was first published on npm on February 6th. And the 1.1.0 release can be dated when releasing.